packages/kbn-dev-utils/src/kbn_node/get_node_bin_dir.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import execa from 'execa';
+import { platform } from 'os';
+import { resolve } from 'path';
+import { REPO_ROOT } from '../repo_root';
+
+// Retrieves the correct location for the current node installation dir
+export function getNodeBinDir() {
+  const nixPrefix = 'bin';
+  const nodeDir = execa
+    .sync('npm', ['config', 'get', 'prefix'], {
```

Comment:
I have verified this works with NVM since it sets the prefix, but are we aware of any other ways folks are installing node which wouldn't account for this? What if you manually install Node?

Reply:
It should not be specific to nvm but to npm. As long as you have node installed and available (which everyone should) npm config prefix should point to where the `bin/node` is located. I just tried that in a local ubuntu vm after installed node both with `apt install nodejs` and also from the basic binaries following https://github.com/nodejs/help/wiki/Installation and the behaviour was also confirmed there. Do you think that answers your question?